cmd/gitbase/command/server.go

```diff
@@ -59,6 +59,7 @@ type Server struct {
 	Format        string         `long:"format" default:"git" choice:"git" choice:"siva" description:"Library format"`
 	Bucket        int            `long:"bucket" default:"2" description:"Bucketing level to use with siva libraries"`
 	Bare          bool           `long:"bare" description:"Sets the library to use bare git repositories, used only with git format libraries"`
+	NonBare       bool           `long:"non-bare" description:"Sets the library to use non bare git repositories, used only with git format libraries"`
```

Comment:
Why is this necessary? Bare == true doesn't mean NonBare == false?

Reply:
By default it autodetects the type (bare or non bare). To be able to know if we explicitly set one of the options we can either use two booleans or an argument with an option `--bare=true`. Using booleans seems more similar to the other options.